examples/mobile/UIComponents/components/list/basiclist.html

```diff
@@ -45,100 +51,57 @@ <h1>
 						</span>
 					</div>
 				</li>
-			</ul>
-			<ul class="ui-listview ui-content-area">
-				<li class="ui-li-divider ui-li-has-icon">
-					<span class="ui-li-icon">
-						<img src="images/thumbnail.jpg"/>
-					</span>
-					<div class="ui-li-text">
-						<span class="ui-li-text-title">
-							Main text 01
-						</span>
-					</div>
-				</li>
 				<li>
-					<span class="ui-li-icon">
-						<img src="images/thumbnail.jpg"/>
-					</span>
 					<div class="ui-li-text">
 						<span class="ui-li-text-title">
-							Main text 02
-						</span>
-						<span class="ui-li-text-sub">
-							It's focused case of icon+text list
+							Main text 04
 						</span>
 					</div>
 				</li>
 			</ul>
 			<div class="ui-content-subheader">Subheader</div>
 			<ul class="ui-listview ui-content-area">
 				<li class="ui-li-divider">
-					<span class="ui-li-text">
+					<div class="ui-li-text">
 						<span class="ui-li-text-title">
-							Sound Mode
-						</span>
-						<span class="ui-li-text-sub ui-li-text-value">
-							Vibration
+							Main text 01
 						</span>
-					</span>
-				</li>
-				<li>
-					<span class="ui-li-text">
-						<span class="ui-li-text-title">
-							Vibration while ringing
+						<span class="ui-li-text-sub">
+							This psd is just a sample. I believe in your possibility. This screen will help.
 						</span>
-					</span>
-					<div class="ui-li-action">
-						<input type="checkbox" class="ui-on-off-switch">
 					</div>
 				</li>
-				<li class="ui-li-subheader"></li>
 				<li class="ui-li-divider">
-					<span class="ui-li-text">
+					<div class="ui-li-text">
 						<span class="ui-li-text-title">
-							Do not distrurb
+							Main text 02
 						</span>
 						<span class="ui-li-text-sub">
-							Mute all calls and alerts, except for your custom exceptions.
+							This is the second line for descriptive phrases. This text color is #909090.
 						</span>
-					</span>
-					<div class="ui-li-divider"></div>
-					<div class="ui-li-action">
-						<input type="checkbox" class="ui-on-off-switch">
 					</div>
 				</li>
-				<li>
-					<span class="ui-li-text">
-						<span class="ui-li-text-title">
-							Ringtone
-						</span>
-					</span>
-				</li>
-				<li class="ui-li-subheader">
-					<span class="ui-li-text-subheader">
-						Subheader
-					</span>
-				</li>
 				<li class="ui-li-divider">
-					<span class="ui-li-text">
+					<div class="ui-li-text">
```

Comment:
I just have a question how to use it.
To use the "ui-li-text-title" and ""ui-li-text-sub", do I have to use "ui-li-text"?
Can I use "ui-li-text-title" directly without "ui-li-text"??

Reply:
'ui-li-text' is the container for texts because title and sub text need to place it vertically (To apply flex-direction: column).
So I define the 'ui-li-text' and include the title and sub. And the container has the padding values, so we need to include it to apply the padding to texts.